Add information about benchcab (#676)
ACCESS-NRI/access-hive.org.au: docs/models/model_components/land.md

<br>
CABLE can also be run as a standalone model, for a single location, a region or globally.

> ### benchcab: evaluation tool for CABLE

Reviewer: ```suggestion
### Benchcab: evaluation tool for CABLE
```

I know _benchcab_ is usually written all lowercase (like _payu_), but for the title in this case I would start with a capital letter.

> <a href="https://benchcab.readthedocs.io/en/stable/" target="_blank">benchcab</a> is a testing framework tool for CABLE. It allows to test CABLE's scientific performance across a range of model configurations and model versions. The tool currently tests CABLE offline only with flux site simulations, using observed eddy covariance data and with spatial simulations over a region or globally. The output of these simulations can then be uploaded to <a href="https://modelevaluation.org/" target="_blank">modelevaluation.org</a> for a statistical analysis of the scientific performance of the supplied configurations.

Reviewer: ```suggestion
(https://benchcab.readthedocs.io/en/stable)[_Benchcab_] is a testing framework tool for CABLE. It allows to test CABLE's scientific performance across a range of model configurations and model versions. The tool currently works with CABLE offline only with flux site simulations, using observed eddy covariance data and with spatial simulations over a region or globally. The output of these simulations can then be uploaded to [modelevaluation.org](https://modelevaluation.org) for a statistical analysis of the scientific performance of the supplied configurations.
```

We are in the process of rewriting everything using Markdown (as much as possible).

Also, I would change the phrasing in:
>The tool currently tests CABLE offline only with flux site simulations, using observed eddy covariance data and with spatial simulations over a region or globally.

I don't know the specifics about `benchcab` so I cannot suggest the appropriate change. 
Is this section meaning that `benchcab` currently works only with CABLE offline, and it can test its performance in 3 main ways: (1) using flux site simulations; (2) using observed eddy covariance data; (3) using spatial simulations? 
"Over a region or globally" is referred to all these 3 methods, or only to the spatial simulations?

> We invite you to refer to <a href="https://benchcab.readthedocs.io/en/stable/" target="_blank">the documentation</a> to know how to use the tool and learn about current limitations to the functionalities.

Reviewer: ```suggestion
To learn more about _benchcab_, its functionalities and limitations, refer to [_benchcab_'s documentation](https://benchcab.readthedocs.io/en/stable/).
```
